rogueutil.h

```diff
@@ -76,6 +76,7 @@
 	#include <sys/ioctl.h> /* for getkey() */
 	#include <sys/types.h> /* for kbhit() */
 	#include <sys/time.h> /* for kbhit() */
+	#include <pwd.h> /* for getpwduid(), getuid() */
```

Comment:
getuid() is in unistd.h on both my GNU/Linux and FreeBSD systems.

Reply:
Sorry, my error. I will correct it.